frontend/src/components/MeetingRoom/Toolbar/Toolbar.tsx

```diff
@@ -152,8 +151,8 @@ const Toolbar = ({
       </div>
       <div className="flex flex-1 justify-center">
         <div ref={mediaControlsRef} className="flex flex-row">
-          <AudioControlButton />
-          <VideoControlButton />
+          <DeviceControlButton isAudioControl />
+          <DeviceControlButton />
```

Comment:
Is this better?

Reply:
https://www.youtube.com/watch?v=_l0f6z2JYOs